python/pyspark/streaming/tests.py

```diff
@@ -659,7 +659,7 @@ def test_kafka_rdd(self):
 
         self._kafkaTestUtils.createTopic(topic)
         self._kafkaTestUtils.sendMessages(topic, sendData)
-
+        self._kafkaTestUtils.waitUntilLeaderOffset(topic, 0, sum(sendData.values()))
```

Comment:
Should be added for the all the kafka unit tests, whereever there is a sendMessage. Its missing in `test_kafka_direct_stream***`